eml/api/api.js

```diff
@@ -9,8 +9,9 @@ const digitalOcean = 'http://207.154.213.68:8888';
 
```

Comment:
I have reimplemented the error handling that was removed. I also removed a console log, and added a comment.